vividus-plugin-web-app/src/main/resources/properties/defaults/run.properties

```diff
@@ -4,3 +4,4 @@ sitemap.parser.follow-redirects=true
 
 selenium.screenshot.shooting-strategy=
 selenium.screenshot.debug-directory=
+desktop-electron-app=false
```

Comment:
it makes sense to put this property under profile, since it's not related to run: `properties/profile/web/profile.properties`